### PR TITLE
Fix Vlan Type, Transceiver , Location and role devices,

### PR DESCRIPTION
**Pull request by gerardocastaldo**
Related to issue #266 

I found and fixed other issues.

### CODE REVIEW

**Review by jdrew82 — COMMENTED** [review]
Just a few questions to address before I can approve this.

File: nautobot_ssot/integrations/aristacv/utils/cloudvision.py [hunk 1]
```diff
@@ -548,8 +548,8 @@ def get_interface_transceiver(client: CloudvisionApi, dId: str, interface: str):
                 "mediaType"
             ):
                 return notif["updates"]["actualIdEepromContents"]["mediaType"]
-            if notif["updates"].get("localMediaType"):
-                return notif["updates"]["localMediaType"]["Name"]
+            if notif["updates"].get("mediaType"):
```
[the remark this answers is not in the document]
Comment (jdrew82) [reply nautobot_ssot/integrations/aristacv/utils/cloudvision.py:551]: Also, if you could provide API examples for unit test fixtures for these changes that'd be appreciated.

[the remark this answers is not in the document]
Comment (gerardocastaldo) [reply nautobot_ssot/integrations/aristacv/utils/cloudvision.py:551]: Hi @jdrew , 

I use the CVaaS and I added the get("mediaType") to identify the fixed 10GE interface for few models (example DCS-7050TX3-48C8). 

Follows I attached few screenshots. 

Here you see like the Ethernet1 dosn't have the actualIdEepromContents field. The Etherne1 is a 10GE fixed interface.

image

Here you see that to identify the interface type we can use the get("mediaType")

image

For Ethernet49, interface with QSFP, we can use whether actualIdEepromContents or the mediaType

image

image

I checked, moreover, that the localMediaType doesn't exists but it's unremovable to pass the checks.

File: nautobot_ssot/integrations/aristacv/diffsync/adapters/cloudvision.py [hunk 2]
```diff
@@ -55,7 +55,9 @@ def load_devices(self):
                 self.add(new_cvp)
             except ObjectAlreadyExists as err:
                 self.job.logger.warning(f"Error attempting to add CloudVision device. {err}")
-        for dev in cloudvision.get_devices(client=self.conn.comm_channel):
+
+        for index, dev in enumerate(cloudvision.get_devices(client=self.conn.comm_channel), start=1):
+            self.job.logger.info(f"Loading {index}° device")
```
Comment (qduk) [inline nautobot_ssot/integrations/aristacv/diffsync/adapters/cloudvision.py:60]: Do we want the `index` or the `dev`here?

Comment (gerardocastaldo) [reply nautobot_ssot/integrations/aristacv/diffsync/adapters/cloudvision.py:60]: Hi @qduk ,
In my architecture, i have 150 devices and during the normal execution i would to know the progress. I insert the index to have a rapid view but it's a good idea to add the device name too.

Comment (qduk) [reply nautobot_ssot/integrations/aristacv/diffsync/adapters/cloudvision.py:60]: Ah I see. Yes, I think having the device name in the log message may be beneficial.

File: nautobot_ssot/integrations/aristacv/utils/cloudvision.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdrew82) [reply nautobot_ssot/integrations/aristacv/utils/cloudvision.py:551]: Upon further review this looks good.

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Pull request merged**